client/src/App.jsx

```diff
@@ -0,0 +1,60 @@
+import { useState } from 'react'
+import { Routes, Route, BrowserRouter, Link } from "react-router"
+import './App.css'
+
+import * as user from './utils/userUtils'
+
+function App() {
+
+  // Handle login/register submission
+  const handleSubmit = (formData, submissionType) => {
+    const username = formData.get("username"), password = formData.get("password")
+
+    submissionType == user.OPTIONS.LOGIN      && user.login(username, password)
+    submissionType == user.OPTIONS.REGISTER   && user.register(username, password)
+
+    if (submissionType != user.OPTIONS.LOGIN && submissionType != user.OPTIONS.REGISTER) {
+      console.error("Invalid type for handleSubmit")
+    }
+  }
+  return (<>
+
+    <header>
+      <h1>Sk8rlog</h1>
+    </header>
+
+    <BrowserRouter>
+      <Routes>
+
+        <Route path='/' element={
+          <form className='landingModal' action={(formData) => handleSubmit(formData, "login")}>
```

Comment:
should we extract these into separate components? (something like `LoginModal` and `RegisterModal`)